packages/models-library/src/models_library/api_schemas_storage.py

```diff
@@ -317,6 +317,7 @@ class FoldersBody(BaseModel):
 
     @model_validator(mode="after")
     def ensure_consistent_entries(self) -> Self:
+        # pylint:disable=no-member
```

Comment:
why do you need this here?

Reply:
correct with source: Annotated[dict[str, Any], Field(default_factory=dict)]
all of them